src/Illuminate/Broadcasting/BroadcastEvent.php

```diff
@@ -14,50 +14,33 @@ class BroadcastEvent implements ShouldQueue
 {
     use Queueable;
 
-    /**
-     * The event instance.
-     *
-     * @var mixed
-     */
-    public $event;
-
     /**
      * The number of times the job may be attempted.
-     *
-     * @var int
      */
-    public $tries;
+    public ?int $tries;
 
     /**
      * The number of seconds the job can run before timing out.
-     *
-     * @var int
      */
-    public $timeout;
+    public ?int $timeout;
 
     /**
      * The number of seconds to wait before retrying the job when encountering an uncaught exception.
-     *
-     * @var int
      */
-    public $backoff;
+    public ?int $backoff;
 
     /**
      * The maximum number of unhandled exceptions to allow before failing.
-     *
-     * @var int
      */
-    public $maxExceptions;
+    public ?int $maxExceptions;
 
     /**
      * Create a new job handler instance.
      *
-     * @param  mixed  $event
      * @return void
      */
-    public function __construct($event)
+    public function __construct(public mixed $event)
     {
-        $this->event = $event;
```

Comment:
This might be a breaking change

Reply:
For the previous versions of PHP?

Reply:
Nope, you removed class property, which might be used in user applications. Class isn't final, so it might be extended.

Reply:
You right. thanks